modules/home/assets/js/components/home-screen.js

```diff
@@ -1,17 +1,24 @@
 import { Container, Box } from '@elementor/ui';
-
 import TopSection from './top-section';
+import PromotionBar from './side-bar-promotion';
```

Comment:
Can the component name and the file name be more the same? `SideBarPromotion` for example

Reply:
fixed